drake/doc/developers.rst

```diff
@@ -43,6 +43,42 @@ available in the superbuild.
 class method runs as a unit test, and fails if it times out or calls
 ``error``. For more information, see ``drake/util/unitTest.m``.
 
+Supported Configurations
+========================
+
+The following table shows the configurations and platforms that Drake
+officially supports. Supported configurations are tested in continuous
+integration. All other configurations are provided on a best-effort basis.
+On Ubuntu and OS X, only makefile builds are supported.
+
++------------------------+--------------------+-------------------+---------+
+| Operating System       | Compiler           | Superbuild Deps   | Build   |
++========================+====================+===================+=========+
+| Ubuntu 14.04 LTS       | GCC 4.9            | Default           | Debug   |
+|                        |                    |                   +---------+
+|                        |                    |                   | Release |
+|                        |                    +-------------------+---------+
+|                        |                    | Default + MATLAB  | Release |
+|                        +--------------------+-------------------+---------+
+|                        | Clang 3.7          | Default           | Debug   |
+|                        |                    |                   +---------+
+|                        |                    |                   | Release |
++------------------------+--------------------+-------------------+---------+
+| Windows Server 2012 R2 | MSVC 14 32-bit     | Default           | Debug   |
```

Comment:
Thanks. I suggest replacing "Windows Server 2012 R2" with "Windows Server 2012 R2 or Windows 8.1".

This may explain why I'm having trouble configuring a Drake development environment using Window 10!